planetwars/client/app/components/about/About.ts

```diff
@@ -1,18 +1,61 @@
 import * as React from 'react';
-import { button, div, h, p } from 'react-hyperscript-helpers';
+import {a, div, h, h1, h2, p} from 'react-hyperscript-helpers';
+import {Link} from "react-router-dom";
+
+// tslint:disable-next-line:no-var-requires
+const styles = require('./About.scss');
 
 interface IProps {
-  counter: number;
-  onIncrement: () => void;
 }
 
-interface IState { }
-
+interface IState {
+}
+// TODO: either make the link to github open externally or add a way to get back to the about page
 export default class About extends React.Component<IProps, IState> {
   public render() {
-    return div([
-      p([this.props.counter]),
-      button({ onClick: () => this.props.onIncrement() }, ['+1']),
+    return div(`.${styles.aboutPage}`, [
+      h1(`.${styles.aboutTitle}`, 'Welcome to PlanetWars!'),
+
+      h2(`.${styles.subTitle}`, 'Getting Started'),
+
+      p(['In order to start a game, you will need at least 2 bots. ' +
+        'A “bot” is what we call the script you will write which will play the game. ' +
+        'For more details on the the rules of PlanetWars and the scripting API, ' +
+        'please refer to ',
+        a({href: 'https://github.com/ZeusWPI/MOZAIC/blob/development/planetwars/README.md'}, ['this guide.']),
+      ]),
+
+      p(['Once you have created your planet-conquering script, you will need to configure it in the client. ' +
+        'In order to do that, go to the ',
+        h(Link, {to: '/Bots'}, ['Bots']),
+        ' page, and enter the required details.']),
+
+      p('Name: surprise, the name of your bot. This is the name which will be used in the game logs, ' +
+        'and which will be shown on the victory screen, so choose well!'),
+
+      p('Command: This is the command you would normally enter in your terminal to run your script. ' +
+        'Make sure all required arguments (such as the filename) are provided here as well.' +
+        'E.g: python3 “~/bots/my_awesome_bot.py”'),
+
+      h2(`.${styles.subTitle}`, 'Playing A Game'),
+
+      p(['With the bots configured as above, head on over to the ',
+        h(Link, {to: '/Play'}, ['Play']),
+        ' page. Here you will see all the bots you have registered. ' +
+        'Select all the bots you would like to enter into battle. To deselect a bot, just click it again.' +
+        'Next, choose the map you would like to play on, or import your own. ' +
+        'Set a turn limit, and finally, hit the Play button.']),
+
+      h2(`.${styles.subTitle}`, 'Visualizing a Game'),
+
+      p(['Once the game has finished playing, you can then load up the gamelog in the visualizer ' +
```

Comment:
This will no longer be the case. it isn't functional yet (should be today), but matches should automatically appear in the matches page.